File: Data Structures & Algorithms/majority-element-ii/submission-0.py
from collections import Counter
class Solution:
    def majorityElement(self, nums: List[int]) -> List[int]:

        l = len(nums)/3

        #nonoptial

        c = Counter(nums)
        res = []


        for i,v in c.items():
            if v > l:
                res.append(i)
        return res





        

File: Data Structures & Algorithms/number-of-one-bits/submission-0.rs
impl Solution {
    pub fn hamming_weight(n: u32) -> i32 {

        let mut res = 0;
        let mut val = n;
        for _ in 0..32{
            if (val&1) == 1 {
                res+=1;
            }
            val >>= 1;
        }
        return res
    }
}
